docs/docs/develop/guest-blockchain-testnet.md

```diff
@@ -0,0 +1,47 @@
+# Guest Blockchain Testnet Guide
```

Comment:
maybe mention that there's no slashing in the first stage?

Reply:
no slashing no rewards also